products/opensuse/main.pm

```diff
@@ -96,7 +96,7 @@ if (check_var('DESKTOP', 'kde') && !get_var('KDE4')) {
 }
 
 # SELinux is now default LSM in enforcing mode (bsc#1230118)
-set_var('SELINUX', 1) if is_tumbleweed && !get_var('SELINUX', 0);
+set_var('SELINUX', 1) if is_tumbleweed && !check_var('SELINUX', 0);
```

Comment:
just thought of one more things to consider

This won't apply in upgrade scenarios from distros where AppArmos is still enabled (i.e from any Leap version).

Reply:
which test scenario are you specifically referring to? can you link me to that?

Reply:
In staging, that would be https://openqa.opensuse.org/tests/4456198

on the full product run, there will be many more (UPGRADE_*, ZDUP_*)

Reply:
can't those be run with SELINUX=0?